classrooms/forms.py

```
@@ -7,7 +7,7 @@ class Meta:
         fields = ['title', 'short_introduction', 'course_description', 
                   'course_image', 'preview_video_link', 'published', 
                   'upcoming', 'published_on', 'paid_course', 
-                  'course_price', 'slug', 'total_rating', 'number_of_ratings']
+                  'course_price', 'total_rating', 'number_of_ratings']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)